fineract-provider/dependencies.gradle

```
@@ -79,7 +79,8 @@ dependencies {
             'io.swagger.core.v3:swagger-annotations',
             'org.webjars:webjars-locator-core',
 
-            'com.google.cloud.sql:mysql-socket-factory-connector-j-8:1.1.0'
+            'com.google.cloud.sql:mysql-socket-factory-connector-j-8:1.1.0',
+            'com.squareup.retrofit2:converter-gson:2.1.0'
```

@fynmanoj sorry I keep being a PITA here, but this `retrofit2:converter-gson:2.1.0` here is not good - you shouldn't fix the version here - because we already fixed a version in the root `fineract/build.gradle` (not `fineract-provider/` - but there we currently have `2.9.0` instead of your old `2.1.0`.. would you be able to just drop the version number here (it should work, and get "inherited", or was there a reason why you NEED `2.1.0` specifically?

```suggestion
            'com.squareup.retrofit2:converter-gson'
```